app/javascript/convert.js

```diff
@@ -0,0 +1,38 @@
+import {saveAs} from "file-saver";
+
+$(() => {
+  $("form#convert").submit((evt) => {
+    evt.preventDefault();
+    let formData = new FormData();
+    formData.set("file", $("input[type=file")[0].files[0]);
+    formData.set("format", $("select").val());
```

Comment:
Prefer to find elements by ID or unique class, so that if we end up adding another `<select>` to the page (e.g. in the navbar) we don't break this page without realizing it.

Reply:
I'll add some IDs to these form elements and refine the jQuery search.